integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -744,7 +744,7 @@ def _collect_assessment_grades_data(self, enterprise_enrollment):
         try:
             assessment_grades_data = self.grades_api.get_course_assessment_grades(course_id, username)
         except HTTPError as err:
-            if err.response.status_code == 404:
+            if err.resposne and err.response.status_code == 404:
```

Comment:
Is this misspelling intentional?
```suggestion
            if err.response and err.response.status_code == 404:
```

Reply:
haha no 🤦